apps/omg_watcher/lib/exit_processor/core.ex

```diff
@@ -359,6 +376,22 @@ defmodule OMG.Watcher.ExitProcessor.Core do
     {%{state | in_flight_exits: Map.merge(ifes, updated_ifes)}, db_updates}
   end
 
+  @spec determine_ife_input_utxos_existence_to_get(ExitProcessor.Request.t(), t()) :: ExitProcessor.Request.t()
```

Comment:
`@doc` - this is very important, as we have "this other function", which does a similar thing, so let's be precise about the behavior of both of them

Reply:
done